contracts/voting-verifier/src/msg.rs

```diff
@@ -68,15 +68,20 @@ pub enum ExecuteMsg {
 }
 
 #[cw_serde]
-pub struct Poll {
-    poll_id: PollId,
-    messages: Vec<Message>,
+pub enum PollData {
+    Messages(Vec<Message>),
+    VerifierSet(VerifierSet),
+}
+#[cw_serde]
+pub struct PollResponse {
+    pub poll: WeightedPoll,
+    pub data: PollData,
```

Comment:
Can we add the poll status? It's computed dynamically now and not stored in WeightedPoll